docs/usage/custom-device-files.md

```diff
@@ -1,12 +1,23 @@
-# Testing custom devices config
+# Using custom devices configuration files
 
-Zwave-js has an iternal database where it stores its device configuration files. Should you wish to modify a device file, or if you need to test a custom config device file, you may do so by placing the file inside a folder in the **store directory** named `config` (By default it is: `<app-root-folder>/store/config`).
+Z-Wave JS has an internal database where it stores its device configuration files. Should you wish to modify a device file, or if you need to test a custom config device file, you may do so by creating a folder inside your store folder named `config` and placing the file inside that folder. (By default it is: `<app-root-folder>/store/config`).
 
-For additional information, see the zwave-js [docs](https://zwave-js.github.io/node-zwave-js/#/api/driver?id=zwaveoptions):
+This directory does not get indexed and should be used sparingly, e.g. when custom files are absolutely necessary or for testing. This directory does not expect any special structure. It can be organized as you like.
 
-> `deviceConfigPriorityDir`: Allows you to specify a directory from where device configuration files can be loaded with higher priority than the included ones. This directory does not get indexed and should be used sparingly, e.g. when custom files are absolutely necessary or for testing.
+For information about the content of device files, see the Z-Wave JS [device file docs](https://zwave-js.github.io/node-zwave-js/#/config-files/file-format).
 
-This directory does not expect any special structure. It can be organized as you like. When a custom device configuration is loaded successfully, you should see this in the zwave-js logs:
+> [!NOTE] ZWavejs2Mqtt needs to be restarted in order to pick up a user-provided device configs. This can be done from settings by hitting `SAVE` or by completely restarting the application.
+> [!NOTE] You must re-interview a device for many types of changes in the device file to have an effect. E.g. changed configuration parameters, certain compat flags, etc.
```

Comment:
Missing line break before the second `> [!NOTE]`

Reply:
line break causes lint issue